src/components/Calculator.js

```diff
@@ -0,0 +1,49 @@
+function Calculator() {
+  return (
+    <div>
+      <Display />
+      <Button />
+    </div>
+  );
+}
+
+function Display() {
+  return (
+    <h2 className="bg-gray color-white text-right pr-1 height-16">0</h2>
+  );
+}
+
+function Button() {
+  const buttons = [
+    { id: 1, value: 'AC', color: 'gray-eo' },
+    { id: 2, value: '+/-', color: 'gray-eo' },
+    { id: 3, value: '%', color: 'gray-eo' },
+    { id: 4, value: '/', color: 'orange' },
```

Comment:
- [ OPTIONAL ] I highly recommend to add `÷` instead of `/` since it is like this in the given image 💯🤩🚀

Reply:
Will do that, Thanks.